backend/globaleaks/tests/utils/test_users_details_filter.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+import json
+from globaleaks.utils.users_details_filter import UserDetailsFilter
+from collections import OrderedDict
+from twisted.trial import unittest
+
+class TestLogUtilities(unittest.TestCase):
+    def test_user_details_filter_removes_pgp_formatted_string(self):
+        mail_body = '''some text before pgp string
+        -----BEGIN PGP PUBLIC KEY BLOCK-----
+        Comment: Alice's revocation certificate
+        Comment: https://www.ietf.org/id/draft-bre-openpgp-samples-01.html
+        iHgEIBYIACAWIQTrhbtfozp14V6UTmPyMVUMT0fjjgUCXaWkOwIdAAAKCRDyMVUM
+        T0fjjoBlAQDA9ukZFKRFGCooVcVoDVmxTaHLUXlIg9TPh2f7zzI9KgD/SLNXUOaH
+        O6TozOS7C9lwIHwwdHdAxgf5BzuhLT9iuAM==Tm8h
+        -----END PGP PUBLIC KEY BLOCK-----
+        some text after pgp string'''
+
+        user_details_filter = UserDetailsFilter(mail_body)
+        expected_formatted_string = 'some text before pgp string\n        *****\n        some text after pgp string'
+        self.assertEqual(user_details_filter.filtered_string(), expected_formatted_string)
+
+        private_pgp_mail_body = '''some text before pgp string
+        -----BEGIN PGP PRIVATE KEY BLOCK-----
+        Comment: Alice's revocation certificate
+        Comment: https://www.ietf.org/id/draft-bre-openpgp-samples-01.html
+        iHgEIBYIACAWIQTrhbtfozp14V6UTmPyMVUMT0fjjgUCXaWkOwIdAAAKCRDyMVUM
+        T0fjjoBlAQDA9ukZFKRFGCooVcVoDVmxTaHLUXlIg9TPh2f7zzI9KgD/SLNXUOaH
+        O6TozOS7C9lwIHwwdHdAxgf5BzuhLT9iuAM==Tm8h
+        -----END PGP PRIVATE KEY BLOCK-----
+        some text after pgp string'''
```

Comment:
Actually the system do not store private keys but only publik keys. I understand the test anyway.

Will be good in a separated patch to get sure that the code that enable to load a key accepts only public keys in first place.

Reply:
Thanks for the comment. I've removed the pgp private key test and also I've changed the regex to the one you suggested.